api/src/task/certificates/setup_cert_user_task.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import uuid
+from datetime import datetime
+from enum import StrEnum
+
+import click
+from sqlalchemy import select
+
+import src.adapters.db.flask_db as flask_db
+import src.util.datetime_util as datetime_util
+from src.adapters import db
+from src.constants.lookup_constants import UserType
+from src.db.models import staging
+from src.db.models.user_models import (
+    Agency,
+    AgencyUser,
+    AgencyUserRole,
+    LegacyCertificate,
+    Role,
+    User,
+)
+from src.task.ecs_background_task import ecs_background_task
+from src.task.task import Task
+from src.task.task_blueprint import task_blueprint
+
+logger = logging.getLogger(__name__)
+
+FUTURE_DATE = datetime_util.make_timezone_aware(datetime(2050, 1, 1), "US/Eastern").date()
```

Comment:
Just make the date directly - making a timezone aware datetime object and then making it a date isn't going to do anything. Dates themselves aren't (generally) timezone aware. The `.date()` function is `return date(self._year, self._month, self._day)` - so the timezone gets ignored.

```suggestion
FUTURE_DATE = date(2050, 1, 1)
```